admin/data/sql/README.md

```diff
@@ -0,0 +1,51 @@
+# SQL Migrations & Seeding
+
+## Creating migrations
+
+- Migrations can be written in Javascript or SQL
+- Javascript migrations must export a generateSql function
+
+Use the create-migration.js tool to create a new migration:
+
+```
+node sql/create-migration.js <name> (default type is sql)
+node sql/create-migration.js <name> --type sql
+node sql/create-migration.js <name> --type js
+```
+Generated migrations have the filename yyyymmddhhmmss.<do|undo>.name.<sql|js>
+
+
+## Running migrations
+
+- The migration script will execute *all* migrations up or down to a given version
+
+```
+node sql/migrate-sql.js
```

Comment:
What is the way to undo all the migrations?
With the existing implementation we could perform that by running `yarn migrate-sql 0`.
Is this already supported?

Reply:
@GeorgeChatzigiannis that's exactly what it says George.  it will execute all migrations up or down to a given version